EPCPortalWeb/wwwroot/lib/jquery/dist/jquery.js

```diff
@@ -1386,7 +1386,7 @@ setDocument = Sizzle.setDocument = function( node ) {
 			return compare;
 		}
 
-		// Calculate position if both inputs belong to the same document
+		// Calculate position if both inputs bedouble to the same document
```

Comment:
typo in comment? Bedouble - noticed it in other comments too, looks like a result of replacing "long" with "double".